mat64/dense.go

```diff
@@ -5,7 +5,10 @@
 package mat64
 
 import (
+	"bytes"
+
 	"github.com/gonum/blas/blas64"
+	"github.com/gonuts/binary"
```

Comment:
I'd really prefer to not add a downstream dependency like this . At the very least we should vendor it. See note below, however.